packages/config/src/layer3s/zklinknexus.ts

```diff
@@ -0,0 +1,42 @@
+import { ProjectId } from '@l2beat/shared-pure'
+
+import { CONTRACTS, TECHNOLOGY, UPCOMING_RISK_VIEW } from '../common'
+import { Layer3 } from './types'
+
+export const zklinknexus: Layer3 = {
+  type: 'layer3',
+  isUpcoming: false,
+  isUnderReview: true,
+  id: ProjectId('zklinknexus'),
+  // TODO(Linea, zkSync Era): zkLink Nexus is a multi-chain zkRollup architecture. In addition to Linea, it currently supports zkSync Era and will support more Layer2s in the future.
+  hostChain: ProjectId('linea'),
```

Comment:
```suggestion
  hostChain: 'Multiple',
```
@maciekzygmunt can you add support for something like this as a short-term solution? we will refine it once we fully review the project